### ISSUE TITLE
Training loss stuck around 0.4

### ISSUE THREAD

**Issue by zzzyzh** [cchen-cc/MA-SAM#22]
I'm using your code for training, but I noticed that the loss doesn't decrease further after a certain point. During training, the loss initially drops to around 0.4, but then it just fluctuates around that value and doesn't go lower. Is this expected behavior? Or could it indicate a potential issue with the training setup?

Any advice would be appreciated. Thanks in advance!

**Comment by mrbird643** [comment]
> I'm using your code for training, but I noticed that the loss doesn't decrease further after a certain point. During training, the loss initially drops to around 0.4, but then it just fluctuates around that value and doesn't go lower. Is this expected behavior? Or could it indicate a potential issue with the training setup?
> 
> Any advice would be appreciated. Thanks in advance!
Could your replication be close to the results presented in the paper?
